pallas-crypto/Cargo.toml

```diff
@@ -18,7 +21,12 @@ rand_core = "0.6"
 pallas-codec = { version = "=0.30.2", path = "../pallas-codec" }
 serde = "1.0.143"
 
+# The vrf crate has not been fully tested in production environments and still has several upstream issues that
+# are open PRs but not merged yet.
+vrf_dalek = { git = "https://github.com/input-output-hk/vrf", rev = "a3185620b72e6a9647285941b961021186f16693" }
```

Comment:
@NicolasDP @AndrewWestberg PR seems to have been merged now.

Regardless, I'm worried that there's no license applied in the upstream repo. Is that something that could be added? Should I open an issue?

Reply:
When I spoke to Sam Leathers, he talked about moving this repository to intersect. I would open it up as an issue and tag him. There's also the KES repository we'd want to do the same thing with.